platform/security/sts/security-sts-ldaplogin/src/main/java/ddf/ldap/ldaplogin/SslLdapLoginModule.java

```diff
@@ -335,6 +337,42 @@ protected boolean doLogin() throws LoginException {
         return true;
     }
 
+    void validateUsername(String username) throws LoginException {
```

Comment:
Another option would be to collapse all of the switch cases together since they do the same thing.